<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>JournalQtClass</class>
 <widget class="QDialog" name="JournalQtClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>429</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>JournalQt</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Nom du ou des auteurs</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>141</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Titre de la référence</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>150</y>
     <width>131</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Nom du journal</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>190</y>
     <width>55</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Volume </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>230</y>
     <width>55</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Numéro</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>270</y>
     <width>55</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Pages</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>310</y>
     <width>55</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Année</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>350</y>
     <width>141</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Identifiant ISSN</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>371</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Entrez les informations de la référence apparue dans le journal</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_auteurJournal">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>60</y>
     <width>181</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_titreJournal">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>100</y>
     <width>181</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_nomJournal">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>140</y>
     <width>181</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_identifiantJournal">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>340</y>
     <width>181</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_anneeJournal">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>300</y>
     <width>181</width>
     <height>27</height>
    </rect>
   </property>
   <property name="maximum">
    <number>3000</number>
   </property>
   <property name="value">
    <number>2019</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_pageJournal">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>260</y>
     <width>181</width>
     <height>27</height>
    </rect>
   </property>
   <property name="maximum">
    <number>1000000000</number>
   </property>
   <property name="value">
    <number>1</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_numeroJournal">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>220</y>
     <width>181</width>
     <height>27</height>
    </rect>
   </property>
   <property name="maximum">
    <number>3000</number>
   </property>
   <property name="value">
    <number>0</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_volumeJournal">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>180</y>
     <width>181</width>
     <height>27</height>
    </rect>
   </property>
   <property name="maximum">
    <number>3000</number>
   </property>
   <property name="value">
    <number>0</number>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_ajouterJournal">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>380</y>
     <width>80</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Ajouter </string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
